readme/usage.md

```diff
@@ -13,10 +13,10 @@ as shown in the screenshots below.
   <img src="usage_2.png"/>
 </p>
 
-In the text input field, type the key to which you would like to map this input.
+In the text input field (right-side, output section), type the key to which you would like to map this input (left-side, input section).
```

Comment:
```suggestion
In the "Output" textbox on the right, type the key to which you would like to map this input.
```

Reply:
Sorry, it took some time. This is my first time going through this `Review-suggestion` workflow. So looked up some docs and videos.


I like your suggestion. How do i proceed, do i just use `Add suggestion to bach` and commit them? But this will create two commits on this pull request which is just a docs change. So i think one commit should be all we need. so do i pull, squash, force-push ?

Reply:
Don't worry, 2 days is certainly fast enough.

I'll squash the commits when I merge the PR, so there is nothing to do wrong